fastplotlib/graphics/_features/_image.py

```diff
@@ -20,6 +20,9 @@ def __init__(self, data, isolated_buffer: bool = True):
 
         data = self._fix_data(data)
 
+        shared = pygfx.renderers.wgpu.get_shared()
+        self._texture_limit_2d = shared.device.limits["max-texture-dimension2d"]
+
```

Comment:
@clewis7 so now instead of tiling textures based on the default 8192 WGPU limit, it will read the limits currently set on the adapter and use those :smile: . This way we can set it to whatever we want in the tests, such as 1024 or 2048, and it will dynamically read the value that has been set for creating the texture array!

Reply:
cool!